### ISSUE TITLE
Can't install Otomi on OVHcloud, because cluster-autoscaler is already installed

### ISSUE THREAD

**Issue by Linutux** [redkubes/otomi-core#871]
The installation on OVHcloud fails, because cluster-autoscaler is installed by default in all installs by their "Managed Kubernetes Service". Otomi fails to install, because "cluster-autoscaler" obviously doesn't have the correct labels set. 

The error is:

> Error: rendered manifests contain a resource that already exists. Unable to continue with install: ClusterRole "cluster-autoscaler" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "cluster-autoscaler"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "kube-system"

**To Reproduce**

1. Add a new cluster in OVHcloud management panel
2. Apply Otomi install

**Expected behavior:** (a clear and concise description of what you expected to happen)
Otomi installs

**Screenshots:** (if applicable, add screenshots to help explain your problem)

**Cluster(s):**

- Cloud: OVHcloud

**Versions:** 

- Kubernetes Version:  1.23.6-2

**Comment by srodenhuis** [comment]
The cluster-autoscaler by default is installed when using the 'custom' provider. If you would like Otomi NOT to install the cluster-autoscaler, you can disable it in the Otomi chart values like this:

```my-values.yaml
cluster:
  k8sVersion: "1.23"
  name: dev
  provider: custom
apps:
  cluster-autoscaler:
    enabled: false
```

and then install the chart:

```
helm repo add otomi https://otomi.io/otomi-core
helm repo update
helm install -f my-values.yaml otomi otomi/otomi
```

I successfully tested this on a managed Kubernetes cluster, with:
- a nodepool with 2 B2-15 nodes and autoscaling enabled
- K8s version 1.23.6-2 in region DE1
- Public network

**Comment by j-zimnowoda** [comment]
Hi @Linutux,
Thanks for your valuable input. Sander has provided a workaround.
We also discussed a possible long-term solution for that issue and decided to create a new `cluster.provider` that will take into account OVHcloud k8s cluster specifics. 
Cheers

**Comment by srodenhuis** [comment]
Solved in #878
